### PR TITLE
Removed view button in alerts and added functionality to row click

### LINKED ISSUE #99: UI : Remove "Eye Icon" from Actions Column in Alert Rules Table
# Description:
The current implementation displays a detailed overview of a specific alert when the user clicks on the "eye icon" in the actions column. However, we have the same functionality when the user clicks on the specific alert row. Therefore, we want to remove the "eye icon" from the actions column.

# Steps to Reproduce:

1. Navigate to the Alerting screen.
2. Add a new alert rule.
3. Observe the alert rules table; the created alert is displayed with an "eye icon" in the actions column.
4. Click on the "eye icon" to view the detailed overview.

# Screenshot: 
image

# Note: 
Remove the "eye icon" functionality from the actions column for newly created alerts. Ensure that the detailed overview functionality is still accessible by clicking on the specific alert row.

### PR DESCRIPTION
**Pull request by Sriya03**
# Description
The view button in the alerts ection for each row is removed

Fixes #99 

# Testing
I have tested it locally

### CODE REVIEW

File: static/js/all-alerts.js [hunk 1]
```diff
@@ -57,27 +57,19 @@ function getAllAlerts(){
 
 class btnRenderer {
 	init(params) {
-		this.eGui = document.createElement('span');
+        this.eGui = document.createElement('span');
```
Comment (nkunal) [inline static/js/all-alerts.js:60]: indentation is not right

Comment (Sriya03) [reply static/js/all-alerts.js:60]: Issue resolved

**Comment by nkunal** [comment]
@sonamgupta21 @sunitakawane please review this.
@Sriya03 please update your branch

**Comment by Sriya03** [comment]
@nkunal Updated branch

**Review by sonamgupta21 — APPROVED** [review]

**Pull request merged**